test/unit/model/test_fallback.py

```diff
@@ -20,7 +20,7 @@ def test_fallback_happy(self):
         forc_section = TestData.load("fallback_index.csv")
 
         fallback_forecast = generate_fallback(
-            forecast_input=forc_section.index,
+            forecast_input=forc_section,
```

Comment:
General comment about unit test naming. What do you think about using the conventions `test_<unit-under-test>_[<test-variant>]`. There are already quite a couple test which apply this convention. So in this case I would suggest to name the tests:

* `test_generate_fallback_happy_flow`
* `test_generate_fallback_empty_load`
* `test_generate_fallback_not_implemented_fallback_strategy`

Or something similar.

Reply:
Good idea, I added it